docs/notebooks/installation.rst

```diff
@@ -1,77 +1,167 @@
+Installing neptune-notebooks 
+============================
+This page provides instructions for installing neptune-notebooks -- the Neptune Jupyter extension. 
 
-Jupyter
--------
-Install `neptune-notebooks extension <https://github.com/neptune-ai/neptune-notebooks>`_:
+The extension integrates between Neptune and Jupyter or JupyterLab. When the extension is successfully installed, 
+you will be able to upload notebooks to Neptune, check out previously uploaded Notebooks, 
+and track experiments and metrics directly from the Jupyter UI. 
+Neptune versions the Notebook automatically once an experiment has started.
+
+About neptune-notebooks
+^^^^^^^^^^^^^^^^^^^^^^^
+
+neptune-notebooks is a Python package hosted `here <https://pypi.org/project/neptune-notebooks>`_. It includes:
+
+- A very basic CLI (command line interface) for uploading notebooks.
+- Standard Jupyter extension. The extension is written in Javascript and is visible in the web-based Jupyter UI. It is not hosted anywhere separately, only as part of the python package.
+- JupyterLab extension. The extension is written in Javascript and is visible in the web-based Jupyter UI. 
+It is hosted `here <https://www.npmjs.com/package/neptune-notebooks>`_.
+
+
+For Jupyter
+-----------
+
+1. Install `neptune-notebooks extension <https://github.com/neptune-ai/neptune-notebooks>`_:
 
 .. code-block:: bash
 
    pip install neptune-notebooks
 
-Then enable extension for your Jupyter:
+2. Enable the extension for Jupyter:
 
 .. code-block:: bash
 
    jupyter nbextension enable --py neptune-notebooks
 
-Remember to install Neptune-client, if you did not do so already:
+3. Install Neptune client, if you did not do so already:
 
 .. code-block:: bash
 
    pip install neptune-client
 
-JupyterLab
-----------
 
-Requirements
-^^^^^^^^^^^^
-`Node.js <https://nodejs.org/en>`_ and `npm <https://www.npmjs.com/get-npm>`_ are required.
+For JupyterLab
+---------------
 
-To check if you have Node.js installed, run this command in your terminal:
+JupyterLab is the next-generation web-based UI for Project Jupyter. 
 
-.. code-block:: bash
+When you install JupyterLab using ``pip install jupyterlab``, the standard Jupyter is also installed, as a dependency.
+
+
+Installation methods
+^^^^^^^^^^^^^^^^^^^^
+You can install the extension using any of the following three methods:
+
+- Through the command line (full neptune-notebooks package)
+- Through the command line (JupyterLab extension only)
+- Using the Extension Manager (JupyterLab extension only)
+
+**Requirements**
+
+Irrespective of which method you use, the following must be preinstalled on your system before you begin.
+
+- `Node.js <https://nodejs.org/en>`_ 
+- `npm <https://www.npmjs.com/get-npm>`_
+- `JupyterLab <https://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html>`_
+
+.. tip:: When installing Python packages, it is best practice to work in a `virtual environment <https://virtualenv.pypa.io/en/latest/>`_. 
```

Comment:
This tip is just as true with regard to installing Standard Jupyter extension. It seems out of place here - there's no `pip install` near it.

Reply:
I added it to the Jupyter section too.
I thought it best to put it high up in the procedures - that is when you should make a decision to go for a venv or not.